packages/common/amplify_db_common/ios/amplify_db_common.podspec

```diff
@@ -0,0 +1,20 @@
+#
+# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html.
+# Run `pod lib lint amplify_db_common.podspec` to validate before publishing.
+#
+Pod::Spec.new do |s|
+  s.name             = 'amplify_db_common'
+  s.version          = '0.0.1'
+  s.summary          = 'A new Flutter plugin project.'
+  s.description      = <<-DESC
+A new Flutter plugin project.
+                       DESC
+  s.homepage         = 'http://example.com'
+  s.license          = { :file => '../LICENSE' }
+  s.author           = { 'Your Company' => 'email@example.com' }
+
```

Comment:
Should these fields be updated?

Reply:
I think the git change was weird, the file was actually moved but git registered it as modified.

Reply:
Updated.